src/main/kotlin/Main.kt

```diff
@@ -1,3 +1,98 @@
-fun main(args: Array<String>) {
-    println("Hello World!")
-}
\ No newline at end of file
+fun main() {
+
+    var mainMunu = 0 //0 - archive, 1 - note, 2 - text note
+    val arhives: MutableList<Storage> = mutableListOf() //тут архивы
+    var activeList = arhives
+    var prevMenu: MutableList<Storage> = mutableListOf()
```

Comment:
```suggestion
    val prevMenu: MutableList<Storage> = mutableListOf()
```
Если сам список мьютабельный, то переменную можно не делать мьютабельной